aspnet/fundamentals/localization/sample/Controllers/AccountController.cs

```diff
@@ -0,0 +1,467 @@
+﻿using System;
```

Comment:
The membership system is used  in the  DataAnnotations localization section to show error messages. Many/most of our tutorials use the template generated code.